src/tests/ftest/util/security_test_base.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/python
+"""
+  (C) Copyright 2020 Intel Corporation.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+  GOVERNMENT LICENSE RIGHTS-OPEN SOURCE SOFTWARE
+  The Government's rights to use, modify, reproduce, release, perform, display,
+  or disclose this software are subject to the terms of the Apache License as
+  provided in Contract No. B609815.
+  Any reproduction of computer software, computer software documentation, or
+  portions thereof marked with this legend must also reproduce the markings.
+"""
+
+import os
+import re
+import random
+from general_utils import pcmd
+
+def acl_entry(usergroup, name, perm, permissions):
```

Comment:
Is this file all code that is also used by the pool ACL tests? If so, don't those existing tests require changes as well? At the very least to remove duplication and import the new file. Perhaps I've missed something.

Reply:
You are totally right, I missed it.
Just did the updates and are now being used by the pool ACL tests. I'm including the changes in the next commit. Thank you!